src/gsi/general_commvars_mod.f90

```diff
@@ -49,6 +50,7 @@ module general_commvars_mod
    public :: s2g_cv                  !  structure used in bkerror.f90
    public :: s2g2                    !  structure used in getprs.f90
    public :: s2g4                    !  structure used in get_derivatives2.f90
+   public :: s2g4x                   !  structure used in get_derivatives2.f90
```

Comment:
This variable : s2g4x , seemingly, is not used in the current GSI,though it is defined. Any planed changes using it?

Reply:
It is used in the updated version of the same subroutine.

Reply:
@jderber-NOAA  . still a little confusing to me.  After updating my local copy, s2g4x is only used in " call general_sub2grid_destroy_info" after it is defined in "call general_sub2grid_create_info". It doesn't have any interactions with other variables/processes.  Then, what is the usage of this variable?  Did I still miss something?

Reply:
Ting,

You are correct.  My mistake for looking at this too quickly.  Thanks for pointing this out.  I have removed all references to s2g4x.